openstack_controller/datadog_checks/openstack_controller/metrics.py

```diff
@@ -535,10 +535,14 @@
 GLANCE_METRICS_PREFIX = "openstack.glance"
 GLANCE_SERVICE_CHECK = f"{GLANCE_METRICS_PREFIX}.api.up"
 GLANCE_RESPONSE_TIME = f"{GLANCE_METRICS_PREFIX}.response_time"
-GLANCE_IMAGE_COUNT = f"{GLANCE_METRICS_PREFIX}.image.count"
+GLANCE_IMAGE_PREFIX = f"{GLANCE_METRICS_PREFIX}.image"
+GLANCE_IMAGE_COUNT = f"{GLANCE_IMAGE_PREFIX}.count"
+GLANCE_IMAGE_METRICS = {f"{GLANCE_IMAGE_PREFIX}.up": {}, f"{GLANCE_IMAGE_PREFIX}.size": {}}
```

Comment:
Can you format "one metric per line" as the rest of metrics?

Reply:
Done (note that the ddev format command `ddev test openstack_controller -fs` makes both of the metrics appear in one line)